modules/subscriptions/views.php

```diff
@@ -150,6 +150,12 @@ static function render_widget_status_messages( $instance ) {
 							__( 'Manage your email preferences.', 'jetpack' )
 						); ?></p>
 					<?php break;
+				case 'many_pending_subs' : ?>
+					<p class="error"><?php printf( __( 'You already have several pending email subscriptions. <br /> Approve or delete a few subscriptions at <a href="%1$s" title="%2$s" target="_blank">subscribe.wordpress.com</a> before continuing.', 'jetpack' ),
```

Comment:
Since you're using HTML markup in that translateable string, I would recommend using `wp_kses` to make sure the string does not include any unwanted HTML.